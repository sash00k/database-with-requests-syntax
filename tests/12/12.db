{"name" : "Иванов С. В.", "birthday" : "10.01.2003", "id" : 1522347612, "department" : "ММФ", "group" : 22129, "student_id" : 421254}
{"name" : "Петрова И. К.", "birthday" : "15.03.2001", "id" : 8992141190, "department" : "ФФ", "group" : 19301, "student_id" : 235251}
{"name" : "Медведев Т. С.", "birthday" : "31.12.2002", "id" : 124556132, "department" : "ФЕН", "group" : 19402, "student_id" : 715541}
{"name" : "Зуброва Д. А.", "birthday" : "10.04.1991", "id" : 2952146170, "department" : "ММФ", "group" : 8126, "student_id" : 52341}
{"name" : "Иванова В. В.", "birthday" : "10.01.2003", "id" : 1522347647, "department" : "ММФ", "group" : 22129, "student_id" : 421255}
{"name" : "Кузнецов И. П.", "birthday" : "25.01.1990", "id" : 5555, "department" : "ММФ", "group" : 8126, "student_id" : 125112}
{"name" : "Кузнецова Л. И.", "birthday" : "20.04.1980", "id" : 4215161, "course" : "Программирование", "groups" : [19129, 8126], "students" : [421254, 124412, 715541]}
{"name" : "Кузнецов И. П.", "birthday" : "20.04.1950", "id" : 1241511124, "course" : "Программирование", "groups" : [19402, 22129], "students" : []}
