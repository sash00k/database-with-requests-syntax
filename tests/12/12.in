get records where ((name is "Кузнецов И. П." and course is set) or department is "ФФ")