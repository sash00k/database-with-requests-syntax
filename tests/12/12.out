Student Петрова И. К. from group 19301, ФФ
Teacher Кузнецов И. П. on course Программирование. Works with 2 groups and 0 students.